covfee/client/tasks/action_annotation/index.tsx

```diff
@@ -42,38 +47,47 @@ const ActionAnnotationTask: React.FC<Props> = (props) => {
   // and we render the component
   return (
     <form>
-      <style jsx>{`
-        button:disabled {
-          background-color: darkorange;
-        }
-      `}</style>
-      <div>
-        <button
-          id="speaking"
-          value="speaking"
-          onClick={() => dispatch(actions.setActiveAnnotation("speaking"))}
-          disabled={active_annotation == "speaking"}
-        >
-          Speaking
-        </button>
-      </div>
-      <div>
-        <button
-          id="laughing"
-          value="laughing"
-          onClick={() => {
-            dispatch(actions.setActiveAnnotation("laughing"));
-          }}
-          disabled={active_annotation == "laughing"}
-        >
-          Laughing
-        </button>
+      <div className={styles.action_annotation_task}>
+        <div className={styles.sidebar}>
+          <AnnotationTypeDropdown
+            annotation_types={annotation_types}
+            itemClick={(item_key: string) =>
+              dispatch(actions.setActiveAnnotation(item_key))
+            }
+            selected_annotation={active_annotation}
+          ></AnnotationTypeDropdown>
```

Comment:
I think this can be simplified as
```suggestion
          >
```